float64[] joint_pos
